_harp/apply/_apply.md

```diff
@@ -2,7 +2,9 @@
 
   Our programmes in Nazareth and London run three times a year, starting in February, June and October.
 
-  Applications for our Summer 2017 programmes open in March.
+  We will be accepting applications for our Summer 2017 programme from Sunday 5th March 2017. Applications for the London cohort close on March 31st 2017. Applications for the Nazareth cohort close on April 30th 2017.
+
+  The summer programme will run from 26th June until 13th October 2017.
```

Comment:
nice idea @jsms90 but i don't feel up to adding in the necessary CSS to make the table look good today. And it's kind of pressing. 
Maybe we can do a simple list instead of a table?